shell.nix

```diff
@@ -70,5 +69,6 @@ mkShell' {
     export PATH=$GOBIN:$PATH
     # install gotestloghelper
     go install github.com/smartcontractkit/chainlink-testing-framework/tools/gotestloghelper@latest
+    go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.57.2
```

Comment:
To make it clear to other people: it's a temporary solution. The proper solution is to upgrade `nix` lock files but we can't do that right now because of Geth versions incompatibility